library/Translate/src/Juvix/ToCore/Types.hs

```diff
@@ -85,6 +85,7 @@ data Error primTy primVal
     UnexpectedOmega
   deriving (Eq, Generic)
 
+-- FIXME replace with PrettyText
 instance (Show primTy, Show primVal) => Show (Error primTy primVal) where
```

Comment:
Is this left for a separate PR?

Reply:
Yeah, exactly, since the FromFrontend errors need pretty-printing for frontend syntax as well